src/resources/ApiKeys/ApiKeys.ts

```diff
@@ -0,0 +1,24 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ApiKeyModel} from './ApiKeysInterfaces';
+
+export default class ApiKey extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/apikeys`;
+
+    list() {
+        return this.api.get<ApiKeyModel[]>(ApiKey.baseUrl);
+    }
+
+    toggle(apiKey: ApiKeyModel, options?: {}) {

```

Comment:
```suggestion
    toggle(apiKey: ApiKeyModel) {
```
If there are no options defined by the API, then there is no need for an `option` parameter